README.md

```diff
@@ -209,8 +209,36 @@ resp, err := rmqc.DeleteQueue("/", "a.queue")
 
 ### Operations on Bindings
 
-TBD
+``` go
+bs, err := rmqc.ListBindings()
+// => []BindingInfo, err
+
+// list bindings in a vhost
+bs, err := rmqc.ListBindingsIn("/")
+// => []BindingInfo, err
+
+// list bindings of a queue 
+bs, err := rmqc.ListQueueBindings("/", "a.queue")
+// => []BindingInfo, err
+
+// declare a binding
+resp, err := rmqc.DeclareBinding("/", "a.binding", BindingInfo{
+	Source: "an.exchange", 
+	Destination: "a.queue", 
+	DestinationType: "queue", 
+	RoutingKey: "#",
+})
+// => *http.Response, err
 
+// deletes individual binding
+resp, err := rmqc.DeleteBinding("/", "a.binding", BindingInfo{
```

Comment:
Looks like this is out of sync with the implementation that takes only 2 arguments now? (as it should, bindings don't have names and are identified by {source name, destination type, destination, routing key, extra arguments})